devel/ansible/roles/dev/files/.bashrc

```diff
@@ -15,6 +15,7 @@ alias brestart="sudo systemctl restart bodhi && echo 'The Application is running
 alias bstart="sudo systemctl start bodhi && echo 'The Application is running on http://localhost:6543'"
 alias bstop="sudo systemctl stop bodhi"
 alias bteststyle="pushd /home/vagrant/bodhi && nosetests -sx ~/bodhi/bodhi/tests/test_style.py; popd"
+alias bfedmsg="sudo journalctl -u fedmsg-tail -f"
```

Comment:
If you leave the ```-f``` off, it still allows the user to pass the ```-f```, but also allows them to see the history. This is what I did with ```blog```.